src/avram/save_operation.cr

```diff
@@ -264,26 +266,6 @@ abstract class Avram::SaveOperation(T) < Avram::Operation
     {% end %}
   end
 
-  macro finished
-    # After all before_save callbacks run, make sure non-nilable column attributes
-    # are there.
-    #
-    # This must run after the other before_* callbacks otherwise it might mark
-    # a field as missing and then later another callback sets the field.
-    #
-    # For example:
-    #
-    #   before_save { email.value = "default@example.com" }
-    #
-    # If we had already validate 'email' it would say 'email is missing' when
-    # in reality We set it in another callback.
```

Comment:
I think moving this bit of comments to the new spot would still be helpful so people know why it is there and not in a regular `before_save`